assets/slider.js

```diff
@@ -47,7 +48,7 @@ class SliderComponent extends HTMLElement {
 
   onButtonClick(event) {
     event.preventDefault();
-    const slideScrollPosition = event.currentTarget.name === 'next' ? this.slider.scrollLeft + this.sliderItems[0].clientWidth : this.slider.scrollLeft - this.sliderItems[0].clientWidth;
+    const slideScrollPosition = event.currentTarget.name === 'next' ? this.slider.scrollLeft + this.sliderItemsToShow[this.sliderItemsToShow.length - 1].clientWidth : this.slider.scrollLeft - this.sliderItemsToShow[this.sliderItemsToShow.length - 1].clientWidth;
```

Comment:
Small nitpick. I see the last item being referenced in 3 places. Can we alias this in the constructor so this line is more readable? Something like `this.lastItem = this.sliderItemsToShow[this.sliderItemsToShow.length - 1]`

Reply:
The reason is because when switching variants, it will put the feature image of that variant first in line. Because it has a `display: none;` before it send it to the first place, the JS sometimes tries to calculate the width of the element that is yet to be shown. I think it happens in terms of milliseconds and doing the last seemed safer in that sense.